dags/mozfun.py

```diff
@@ -0,0 +1,24 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from utils.gcp import gke_command
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2020, 6, 11),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+with DAG("mozfun", default_args=default_args, schedule_interval="0 1 * * *") as dag:
```

Comment:
The `schedule_interval` could also be more frequently, but maybe once a day is also alright for now.

Reply:
Once a day sounds good, and we can clear the most recent task in the UI manually to trigger a mid-day update.